View.lua

```diff
@@ -72,16 +74,26 @@ local function batchsize(input, size, numInputDims, numElements)
 end
 
 function View:updateOutput(input)
-   local bsz = batchsize(input, self.size, self.numInputDims, self.numElements)
-   if bsz then
-      self.output = input:view(bsz, table.unpack(self.size:totable()))
-   else
-      self.output = input:view(self.size)
-   end
-   return self.output
+  if not input:isContiguous() then
+    self._input:resizeAs(input):copy(input)
```

Comment:
Saved models doesn't have this `_input` field, so you should lazily initialize it.